src/web-ui/src/public/Main.vue

```diff
@@ -181,11 +184,13 @@ export default {
       if (response.headers) {
         if (response.headers['x-personalize-recipe']) {
           this.personalized = true
-          this.explain_recommended = 'Personalize recipe: ' + response.headers['x-personalize-recipe']
+          this.recipe = response.headers['x-personalize-recipe']
+          this.explain_recommended = 'Personalize recipe: ' + this.recipe
         }
         if (response.headers['x-experiment-name']) {
           this.active_experiment = true
-          this.explain_recommended = 'Active experiment: ' + response.headers['x-experiment-name']
+          this.experiment = response.headers['x-experiment-name']
```

Comment:
Is `this.experiment` needed? Doesn't look like it's used anywhere besides the next line.